recipes/cluster.rb

```diff
@@ -25,21 +25,20 @@
 cluster_nodes = node['rabbitmq']['clustering']['cluster_nodes']
 cluster_nodes = cluster_nodes.to_json
 
+auto_cluster_nodes = cluster_nodes
+static_cluster_nodes = cluster_nodes
+
 # Manual clustering
 unless node['rabbitmq']['clustering']['use_auto_clustering']
   # Join in cluster
-  rabbitmq_cluster cluster_nodes do
+  rabbitmq_cluster auto_cluster_nodes do
```

Comment:
Maybe Instead of creating those two variables, create a unique name for the rabbitmq_cluster resource, e.g.
```ruby
rabbitmq_cluster "Join Cluster (#{node['rabbitmq']['clustering']['cluster_name']})" do
  cluster_nodes cluster_nodes
  cluster_name node['rabbitmq']['clustering']['cluster_name']
  action :join
end
```